lib/options.js

```diff
@@ -45,17 +45,25 @@ function compute(processArgv) {
     alias: Flags.flags
       .filter((flag) => flag.alias)
       .reduce((object, flag) => {
-        object[flag.name] = flag.alias;
+        if (flag.alias) {
+          object[flag.name] = flag.alias;
```

Comment:
There has to be a way to tell TS that `flag.alias` is always false, right? :thinking:

Reply:
Is this what you mean? https://github.com/total-typescript/ts-reset#make-filterboolean-filter-out-falsy-values

Reply:
Yes, that's what I meant.  I don't know how smart `ts-reset` makes `filter` in this case, but I'd imagine that in this case TS will not remember that `flag.alias` is non-falsy. If it does, then yes, otherwise, it's so close :grin:

Reply:
Fixed it locally, I tossed it into a gist:
https://gist.github.com/lishaduck/8cabda9ae913535f702a4376712d4608

I'd just give a patch, but I'm tweaking some other types. I'll PR soon (unless I forget 😉)